src/patches/Slot.js

```diff
@@ -31,6 +32,54 @@ export const SlotPatches = utils.getOwnPropertyDescriptors({
           nodeData.assignedNodes) || []) :
         [];
     }
+  },
+
+  /**
+   * @this {HTMLSlotElement}
+   * @param {string} type
+   * @param {Function} fn
+   * @param {Object|boolean=} optionsOrCapture
+   */
+  addEventListener(type, fn, optionsOrCapture) {
```

Comment:
`SlotPatches.addEventListener` will be added to the `Element`'s prototype when running on browsers where `HTMLSlotElement` is not available, e.g. Edge/IE11.
https://github.com/webcomponents/shadydom/blob/e953b75ba4c62f207c9b617e1af72af8a91899a8/src/patch-prototypes.js#L57-L59

This could be an issue.

Reply:
Yeah, verified it's not working on Edge. See [tests](https://travis-ci.org/webcomponents/shadydom/builds/500117823?utm_source=github_status&utm_medium=notification)